ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueConsumption.scala

```diff
@@ -76,18 +77,28 @@ object KeyValueConsumption {
 
       case DamlLogEntry.PayloadCase.CONFIGURATION_ENTRY =>
         val configEntry = entry.getConfigurationEntry
-        val newConfig = parseDamlConfiguration(configEntry.getConfiguration).get
-        List(Update.ConfigurationChanged(configEntry.getSubmissionId, newConfig))
+        val newConfig = Configuration.decode(configEntry.getConfiguration).right.get
```

Comment:
hashtag yolo

Reply:
thanks, will address my FIXME and make the function fail properly. Do you think `Try[List[Update]]` is a reasonable API for this?

Reply:
hm, `Either[String, List[Update]]` seems to fit it better.